Packages/com.llealloo.audiolink/Runtime/Scripts/YtdlpPlayer.cs

```diff
@@ -167,7 +167,7 @@ public static void LocateYtdlp()
 
             // Offer to download yt-dlp to the AudioLink folder ONLY when we are not in VRC usecase,
             // since VRC's security team doesn't like us downloading executables.
-            #if !VRC_SDK_VRCSDK3
+            #if !VRC_SDK_VRCSDK3 && UNITY_EDITOR_WIN
```

Comment:
the cynic in me wants to find a Mac user to test this with

Reply:
but that should be a fraction of our userbase, so if you decide you don't care I'm fine with that

Reply:
I tested it: if you bake a disabled avatar the result will also be disabled

Reply:
wha? did you leave this comment on the wrong PR or something. Whats this about baking

Reply:
anyways, this doesn't worsen the situation on mac, so I think its fine.